modules/runner/src/main/java/org/apache/ignite/internal/app/IgniteImpl.java

```diff
@@ -799,6 +811,7 @@ public CompletableFuture<Ignite> start(Path configPath) {
 
                         return cmgMgr.onJoinReady();
                     }, startupExecutor)
+                    .thenComposeAsync(v -> systemViewManager.completeRegistration())
```

Comment:
if my understanding correct, this is more like temporal solution... If so, let's add comment here describing why do we need this extra step

Reply:
Added TODO)